tests/IntegrationTests/TaskManager.IntegrationTests/Support/Assertions.cs

```diff
@@ -0,0 +1,83 @@
+﻿// SPDX-FileCopyrightText: © 2021-2022 MONAI Consortium
+// SPDX-License-Identifier: Apache License 2.0
+
+using FluentAssertions;
+using Monai.Deploy.Messaging.Events;
+using Monai.Deploy.WorkflowManager.TaskManager.AideClinicalReview.Events;
+
+namespace Monai.Deploy.WorkflowManager.TaskManager.IntegrationTests
+{
+    public class Assertions
+    {
+        public Assertions(ISpecFlowOutputHelper output)
+        {
+            Output = output ?? throw new ArgumentNullException(nameof(output));
+        }
+
+        private ISpecFlowOutputHelper Output { get; set; }
+
+        public void AssertClinicalReviewEvent(ClinicalReviewRequestEvent clinicalReviewRequestEvent, TaskDispatchEvent taskDispatchEvent)
+        {
+            Output.WriteLine("Asserting details of ClinicalReviewRequestEvent with TaskDispatchEvent");
+            clinicalReviewRequestEvent.ExecutionId.Should().Be(taskDispatchEvent.ExecutionId);
+            clinicalReviewRequestEvent.CorrelationId.Should().Be(taskDispatchEvent.CorrelationId);
+            clinicalReviewRequestEvent.TaskId.Should().Be(taskDispatchEvent.TaskId);
+            clinicalReviewRequestEvent.PatientMetadata.PatientId.Should().Be(GetTaskPluginArguments(taskDispatchEvent, "patient_id"));
+            clinicalReviewRequestEvent.PatientMetadata.PatientName.Should().Be(GetTaskPluginArguments(taskDispatchEvent, "patient_name"));
+            clinicalReviewRequestEvent.PatientMetadata.PatientSex.Should().Be(GetTaskPluginArguments(taskDispatchEvent, "patient_sex"));
+            clinicalReviewRequestEvent.PatientMetadata.PatientDob.Should().Be(GetTaskPluginArguments(taskDispatchEvent, "patient_dob"));
+            clinicalReviewRequestEvent.WorkflowName.Should().Be(GetTaskPluginArguments(taskDispatchEvent, "workflow_name"));
+
+            foreach (var file in clinicalReviewRequestEvent.Files)
+            {
+                var taskDispatchFile = taskDispatchEvent.Inputs.FirstOrDefault(x => x.Name.Equals(file.Name));
+
+                file.Name.Should().Be(taskDispatchFile?.Name);
+                file.Endpoint.Should().Be(taskDispatchFile?.Endpoint);
+                // file.Credentials.Should().BeEquivalentTo(taskDispatchFile?.Credentials); - WorkflowManager to pass the credentials
+                file.Bucket.Should().Be(taskDispatchFile?.Bucket);
+                file.RelativeRootPath.Should().Be(taskDispatchFile?.RelativeRootPath);
+            }
+            Output.WriteLine("Details of ClinicalReviewRequestEvent matches TaskDispatchEvent");
+        }
+
+        public void AssertTaskUpdateEventFromTaskCallback(TaskUpdateEvent taskUpdateEvent, TaskCallbackEvent taskCallbackEvent, TaskExecutionStatus status)
+        {
+            Output.WriteLine("Asserting details of TaskUpdateEvent with TaskCallbackEvent");
+            taskUpdateEvent.ExecutionId.Should().Be(taskCallbackEvent.ExecutionId);
+            // BUG taskUpdateEvent.CorrelationId.Should().Be(taskDispatchEvent.CorrelationId);
```

Comment:
There are a few commented out lines in this document. Just checking these are expected?

Reply:
Yeah theres a bug open for this.